Xamarin.Android-Tests.sln

```diff
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Is this a BOM change?  Not sure what changed here.

Reply:
Not sure why Github is reporting this as change, there is no changes on this line.